src/com/facebook/buck/android/AndroidLibraryDescription.java

```diff
@@ -175,5 +176,6 @@ public boolean hasFlavors(ImmutableSet<Flavor> flavors) {
   public static class Arg extends JavaLibraryDescription.Arg {
     public Optional<SourcePath> manifest;
     public Optional<String> resourceUnionPackage;
+    public Optional<String> finalRName;
```

Comment:
The version of BK that has this support is not released yet, but I will add docs with a note